dazel.py

```diff
@@ -41,7 +41,7 @@
 DEFAULT_DOCKER_RUN_PRIVILEGED = False
 DEFAULT_DOCKER_MACHINE = None
 DEFAULT_WORKSPACE_HEX = False
-
+DEFAULT_GPUS=""
```

Comment:
1. Space around the `=`
2. Please place after DEFAULT_ENV_VARS, to be near other similar vars.